clang/test/CodeGen/builtins-wasm.c

```diff
@@ -741,6 +741,12 @@ __externref_t externref_null() {
   // WEBASSEMBLY-NEXT: ret
 }
 
+int externref_is_null(__externref_t arg) {
+  return __builtin_wasm_ref_is_null_extern(arg);
+  // WEBASSEMBLY: tail call i32 @llvm.wasm.ref.is_null.extern(ptr addrspace(10) %arg)
```

Comment:
I suppose the llvm side of this is already working/testing?

Reply:
Yeah @pmatos implemented it four years ago:
https://reviews.llvm.org/D114979